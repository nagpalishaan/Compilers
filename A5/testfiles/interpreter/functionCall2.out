test1 10
test1 5
test1 100
test3: 0
