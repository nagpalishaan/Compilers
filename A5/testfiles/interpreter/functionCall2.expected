10
5
100
0
